views/track.hbs

```diff
@@ -0,0 +1,9 @@
+{{#each track}}
+<div >
+  <h1>Title</h1>
```

Comment:
Here, instead of Title, you could've passed `{{this.name}}` to display each song's name!